samcli/commands/deploy/utils.py

```diff
@@ -63,6 +66,7 @@ def print_deploy_args(
     click.echo(f"\tStack name                   : {stack_name}")
     click.echo(f"\tRegion                       : {region}")
     click.echo(f"\tConfirm changeset            : {confirm_changeset}")
+    click.echo(f"\tDisable rollback             : {disable_rollback}")
```

Comment:
Another question about order. This time for the table printed after initiating the deployment. Is it okay to print this value after confirm changeset?